### PR TITLE
Support GitHub matrix

### PR DESCRIPTION
**Pull request by phracek**
This pull request adds support for GitHub action matrix.

The first one supports only docker-tests and the second one openshift-tests.

It brings also a new feature like RHEL8-OpenShift4 tests.

### CODE REVIEW

**Comment by phracek** [comment]
[test][test-openshift][test-openshift-4]

**Comment by phracek** [comment]
[test]

File: .github/workflows/docker-tests.yml [hunk 1]
```diff
@@ -42,92 +66,84 @@ jobs:
           # Create a JSON file for Testing Farm in order to schedule a CI testing job
```
Comment (phracek) [inline .github/workflows/docker-tests.yml:66]:
```suggestion
          # Create a JSON data structure for GitHub API to display the test status in the PR
```

File: .github/workflows/docker-tests.yml [hunk 2]
```diff
@@ -1,4 +1,4 @@
-name: RHEL7-openshift4-tests@TF
+name: docker-tests at Testing Farm
```
Comment (phracek) [inline .github/workflows/docker-tests.yml:1]:
```suggestion
name: container-tests at Testing Farm
```

File: .github/workflows/docker-tests.yml [hunk 3]
```diff
@@ -7,15 +7,40 @@ on:
 jobs:
   build:
     # This job only runs for '[test]' pull request comments by owner, member
-    name: Schedule test on Testing Farm service for RHEL7 - OpenShift 4
+    name: Docker tests on Testing Farm service
```
Comment (phracek) [inline .github/workflows/docker-tests.yml:10]:
```suggestion
    name: Container tests on Testing Farm service
```

**Comment by zmiklank** [comment]
[test]

**Comment by phracek** [comment]
[test-openshift][test-openshift-4]

**Comment by phracek** [comment]
This pull request does not contain any fixes which need regeneration of code.

It can be merged.

**Pull request merged**